ProcessMaker/Http/Controllers/Process/ScreenController.php

```diff
@@ -95,7 +95,7 @@ public function import(Screen $screen)
      */
     public function download(Screen $screen, $key)
     {
-        $fileName = snake_case($screen->title) . '.json';
+        $fileName = trim($screen->title) . '.json';
```

Comment:
I think the SLUG method would be a better choice.

https://laravel.com/docs/5.7/helpers#method-str-slug

Reply:
The problem with the slug method is that it lowercases the name. So MyProcess will be exported as "myprocess", is this ok?